### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by Gdg9333** [siderakb/ergo-snm-keyboard#14] [Q&A] [answered]
Hello. I'm new to all this but have been wanting a split ergo keyboard with a trackball. Finding this was perfect! Thank you for working on this. As a newbie I wanted to ask two things:

- Do you have a set of tools that would be required for this or recommended for this? 
- Would I be able to use the wireless option or is that not ready?

**Comment by ziteh** [comment, marked as answer]
Hi

- If you are using the PCBA service to order PCBs, you only need a soldering iron (the soldering iron I use is Miniware TS101) and precision screwdrivers. If you are not using PCBA, you may also need a hot air gun or a hot plate in order to solder the QFN RP2040.
- The wireless version (Rev 3.x) is functionally complete ([youtube demo video](https://youtu.be/rq8fAxg5dbY)), but I've never actually assembled it, and the case hasn't been redesigned yet (although it may be backwards compatible). So the wireless is not 100% complete. I probably won't have time to finish it anytime soon.

**Reply by Gdg9333** [reply]
Thank you! 👍
